usagov/usagov-2021
# USAGOV-1969 Manage search result weights

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1969

## Description

Installs the sortableviews module to provide a drag-and-drop UI for managing the weight field of search results.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- On English benefit finder page, should see a local task linking to the sorting view. 
 
![image](https://github.com/user-attachments/assets/c985cf3f-fce5-44e0-8d89-d1dc25060d72)

   - Clicking on the link should take you to the view for sorting English benefit pages. `/admin/manage-benefits-search-sort/en`

![image](https://github.com/user-attachments/assets/b5f200b3-3843-4b70-8f17-aae3a1aab1c5)

    - Change the order of some items (click on handle to activate , then click and drag to new location).
    - Return to the benefit search page and confirm your new order is reflected in the search results. May need a cache refresh to get the latest json file for search order.

- Spanish benefit finder page should have a similar local task that goes to `/admin/manage-benefits-search-sort/en` and affects the order of search results in the same way as above.
- Confirm the local task does not show up on other pages.
- Confirm the local task shows up if you're logged in as a Content Administrator or Site Admin
- Confirm the local task does not show up for other roles.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
